code/game/objects/items/weapons/rpd.dm

```diff
@@ -200,6 +206,44 @@ var/list/pipemenu = list(
 		return
 	SSnanoui.update_uis(src)
 
+//RPD radial menu
+
+/obj/item/rpd/proc/check_menu(mob/living/user)
```

Comment:
You know, looking at this code makes me wonder if radial menus should be made into a component, given that a lot of this code is shared